src/legacy/core_plugins/kibana/index.js

```diff
@@ -178,6 +178,9 @@ export default function (kibana) {
               saved_objects: true,
               index_patterns: true,
             },
+            advancedSettings: {
+              showWriteControls: true
```

Comment:
Can we rename this to `save`, as this is closer to how it's consumed and used in the UI itself?